contrib/kvm-setup/roles/kvm-setup/tasks/user.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Create user {{ k8s_deployment_user }}
+  user:
+    name: "{{ k8s_deployment_user }}"
+    groups: adm
```

Comment:
this group is mandatory and is useful only in ubuntu/debian, but not harmful for centos/fedora/rhel